### PR TITLE
Add maven to the project

### PR DESCRIPTION
**Pull request by alejandrocq**
Issue #5

I have added maven with maven-compiler-plugin and maven-jar-plugin. The Java version is 15 as specified in the repository's README.

I have seen that there is already another PR to add Gradle, so feel free to reject this if you don't need it finally ;).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.mdcdxcvi.spider_arena</groupId>
+    <artifactId>spider_arena</artifactId>
+    <version>0.1-SNAPSHOT</version>
```
Comment (romavini) [inline pom.xml]: Leave just version 1.0, please.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.mdcdxcvi.spider_arena</groupId>
```
Comment (romavini) [inline pom.xml]: Replace tha package name to io.github.mdcdxcvi.

**Comment by romavini** [comment]
Update the `README.md` with the new way of running the project with Maven.

File: pom.xml (hunk 1 above)
Comment (alejandrocq) [reply pom.xml]: Done!

File: pom.xml (hunk 2 above)
Comment (alejandrocq) [reply pom.xml]: Done!

**Comment by alejandrocq** [comment]
> Update the `README.md` with the new way of running the project with Maven.

Done! I have also added the plugin maven-exec-plugin, so you can run the game directly using a maven goal.

**Comment by romavini** [comment]
Hi! Thanks for your contribution! 
^^

**Pull request merged**